prqlc/prql-compiler/src/sql/operators.rs

```diff
@@ -123,20 +123,24 @@ fn find_operator_impl(
     dialect: Dialect,
 ) -> Option<(&pl::Func, Option<i32>, bool, Option<String>)> {
     let operator_name = operator_name.strip_prefix("std.").unwrap();
-
-    let operator_name = pl::Ident::from_name(operator_name);
+    let operator_ident = pl::Ident::from_path(
+        operator_name
+            .split('.')
+            .map(String::from)
+            .collect::<Vec<_>>(),
+    );
```

Comment:
`std.sql.prql` was not meant to have submodules for things other than dialect handling, but it is clearer this way.